blueprints/hpc-cluster-small.yaml

```diff
@@ -1,13 +1,13 @@
 blueprint_name: hpc-cluster-small
 
 vars:
-  billing_account: ## Set GCP Billing Account Here ##
-  org_id: ## Set GCP Organization ID Here ##
-  folder_id: ## Set GCP Folder ID Here ##
-  project_id: ## Set GCP Project ID Here ##
-  deployment_name: ## Set Deployment Name Here ##
-  region: ## Set Region Here ##
-  zone: ## Set Zone Here ##
+  billing_account: 019162-C95952-41672F ## Set GCP Billing Account Here ##
```

Comment:
These values do not need and should not be hardcoded

Reply:
true will remove them